### DISCUSSION TITLE
Is there anything in the beeware/toga ecosystem that's similar to react native?

### DISCUSSION THREAD

**Discussion by richstokes** [beeware/toga#1629] [Q&A]
Curious about this. i.e. is there something that provides the boilerplate/skeleton for basic navigation and iOS/Android UI elements? Handles displaying nice-looking buttons, modals, etc. 

It looks like the UI elements currently implemented are very primitive, so you wouldn't want to use this to actually build an iPhone app, for example? (At least not one that looks reasonably good and *feels* like a proper iPhone app with a native-looking interface)

Am I correct here, or am I missing something?

Thanks!

**Comment by freakboy3742** [comment]
Toga *is* the React Native equivalent; the catch is that the range of widgets that have been implemented, especially on mobile, is sparse and hasn't ventured as much into the "navigation management" area.

However, that is an area we are intending to work on in the near future.
